screens/MenteeDetailsView.js

```diff
@@ -15,9 +16,41 @@ class MenteeDetailsView extends Component {
     headerTitleStyle
   });
 
+  state = {
+    messageDeltaString: '0 days'
+  };
+
+  async componentDidMount() {
+    try {
+      const messageContextJson = await AsyncStorage.getItem('messages');
+      if (messageContextJson) {
```

Comment:
Is this necessary? Wont AsyncStorage throw an error if the getItem fails?

Reply:
That's a good question. I don't know what happens if you push a falsey value to messages though, so we might want to keep the check.

Reply:
It looks like the example code they give uses a check, so thats fine